paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/FakeFormActivityLauncher.kt

```diff
@@ -0,0 +1,48 @@
+package com.stripe.android.paymentelement.embedded
+
+import androidx.activity.result.ActivityResultCallback
+import androidx.activity.result.ActivityResultCaller
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.ActivityResultRegistry
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.core.app.ActivityOptionsCompat
+
+internal class FakeFormActivityLauncher : ActivityResultLauncher<FormContract.Args>() {
+    var didLaunch = false
+        private set
+    var launchArgs: FormContract.Args? = null
+        private set
+    var didUnregister = false
+        private set
+
+    override fun launch(input: FormContract.Args?, options: ActivityOptionsCompat?) {
+        didLaunch = true
```

Comment:
It's probably easier to both read and write these tests using Turbine. We would add to the turbine here and in unregister, and add another method to validate them once the test is over.

Reply:
https://jira.corp.stripe.com/browse/MOBILESDK-3029